src/main/java/com/aspose/imaging/cloud/sdk/invoker/Configuration.java

```diff
@@ -40,7 +40,7 @@
         /**
     	 * The default API version
     	 */
-        public final String DefaultApiVersion = "v2";
+        public final String DefaultApiVersion = "v2.0";
 
```

Comment:
Is it expected change ?

Reply:
@aifeigin minor unification to avoid possible CI problems in the future.